TACC-Cloud/hazmapper
# Hotfix/fix vite config and remove warning

## Overview: ##

This PR has some minor fixes:
* fixes a default prop warning e0354254dc59ba3c464b131448e4dd7bea83033d
* update jest coverage configuration so that we see files that aren't tested
* remove duplicate resolve in react/vite.config.ts bca7249d20f01b538ce713d765636eb958795dc0

## PR Status: ##


## Testing Steps: ##
1. Run unit tests (not coverage)
2. Run app and note that default prop warning is gone (there are still some for the core-components library)
